### ISSUE TITLE
Can't establish connection

### ISSUE THREAD

**Issue by voyager** [gragland/chatgpt-chrome-extension#3]
Hey Gabe,

Thanks for the extension! 🙌
After a lot of fiddling (cursed be Windows!) I managed to get the server.js running
```
✔ Connecting to ChatGPT
✔ Training ChatGPT (5 plugin rules)
✔ You may now use the extension,
```

and the extension installed, but then when I try using the extension via the right-click, I get the error on the screenshot below. The bottom one on the first run, then the top one on every next one.

Do you know what might be causing it?

![000883](https://user-images.githubusercontent.com/457281/206822150-b62d4d33-8bf3-4e16-ba8d-116dd6ba029c.png)

Thanks!

**Comment by gragland** [comment]
Weird it seems to be indicating the extension is installed twice, or at least that a menu item with the same name is installed already. Maybe try deleting the extension and then re-adding it?

**Comment by voyager** [comment]
I did install it into Vivaldi and two Chrome profiles. This latter might've been the issue, if not all 3.

I still got this error below when I removed it from Chromes and reinstalled it in Vivaldi.

![000885](https://user-images.githubusercontent.com/457281/206829674-51f44f1e-a09e-4007-bdf1-9390a0bc28c2.png)

**Then, I opened some online notepad and tested it there and it worked. I was testing it previously in the search fields of the pages I had opened. Apparently, that is not supported. :)** 

Thanks for the quick reply! 🙌
